service/controller.go

```diff
@@ -229,6 +230,9 @@ func (s *service) CreateVolume(
 		sourceVolumeID                    string
 		snapshotIsiPath                   string
 		isROVolumeFromSnapshot            bool
+		isRWVolumeFromSnapshot            bool
+		sourceIsSnapshot                  bool
+		sourceIsVolume                    bool
```

Comment:
can we rename these vars to isSourceSnapshot and isSourceVolume?

Reply:
Renamed.